server/src/helpers/character/CharacterHelper.ts

```diff
@@ -541,7 +541,7 @@ export class CharacterHelper extends BaseService {
     }
 
     // can't move more than one screen at a time
-    character.totalStats[Stat.Move] = clamp(character.totalStats[Stat.Move], 0, 4);
+    character.totalStats[Stat.Move] = clamp(character.totalStats.move ?? 0, 0, 4);
```

Comment:
why was this changed? Stat.Move is valid

Reply:
Ok, changed back